Core/src/main/java/org/tribuo/ensemble/WeightedEnsembleModel.java

```diff
@@ -112,4 +134,78 @@ public WeightedEnsembleModel(String name, EnsembleModelProvenance description, I
     protected EnsembleModel<T> copy(String name, EnsembleModelProvenance newProvenance, List<Model<T>> newModels) {
         return new WeightedEnsembleModel<>(name,newProvenance,featureIDMap,outputIDInfo,newModels,combiner);
     }
+
+    /**
+     * Creates an ensemble from existing models. The model outputs are combined using uniform weights.
+     * <p>
+     * Uses the feature and output domain from the first model.
+     * <p>
+     * If the output domains don't cover the same dimensions then it throws {@link IllegalArgumentException}.
+     * @param name The ensemble name.
+     * @param models The ensemble members.
+     * @param combiner The combination function.
+     * @param <T> The output type.
+     * @return A weighted ensemble model.
+     */
+    public static <T extends Output<T>> WeightedEnsembleModel<T> createEnsembleFromExistingModels(String name, List<Model<T>> models, EnsembleCombiner<T> combiner) {
+        return createEnsembleFromExistingModels(name,models,combiner,Util.generateUniformVector(models.size(), 1.0f/models.size()));
+    }
+
+    /**
+     * Creates an ensemble from existing models. The model outputs are combined using uniform weights.
+     * <p>
+     * Uses the feature and output domain from the first model.
```

Comment:
'Uses the feature domain from the first model' could be read to mean that it excludes features not found in that model when creating the ensemble. Is that true? If not I prefer

```suggestion
     * Uses the output domain from the first model. Though the ensemble itself uses the union 
     * of feature domains from the input models, only the first model's feature domain is recorded
     * in provenance.
```

Reply:
I was leaving a bit of wiggle room here. There's an optimization I'm considering where we add `Prediction<T> predict(SGDVector vector)` to the models and then only pay the vector construction cost (and it's associated hashmap lookups and other things) once. I agree that that might make models created here invalid, and I guess I could lock it down so it throws if the feature domains are different?

Reply:
I've updated the text.